.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: scala
+dist: trusty
```

Comment:
this is fix for travis not to go crazy, see https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/6?u=asarturas